src/bindings/c/include/openvino/c/ov_model.h

```diff
@@ -83,20 +84,22 @@ OPENVINO_C_API(bool) ov_model_is_dynamic(const ov_model_t* model);
  * @param model A pointer to the ov_model_t.
  * @param tensor_names input tensor name (char *) list.
  * @param partialShape A PartialShape list.
- * @param cnt The item count in the list.
+ * @param size The item count in the list.
+ * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
 ov_model_reshape(const ov_model_t* model,
-                 const char* tensor_names[],
-                 const ov_partial_shape_t* partial_shapes[],
-                 size_t cnt);
+                 const char** tensor_names,
+                 const ov_partial_shape_t** partial_shapes,
+                 size_t size);
 
 /**
  * @brief Do reshape in model with partial shape for a specified name.
  * @ingroup model
  * @param model A pointer to the ov_model_t.
  * @param tensor_name input tensor name (char *).
```

Comment:
Why do you add a data type for tensors in the documentation (not only in this place)?

It looks like tensor names have unique data type.

Reply:
ok, let's remove such redundant contents.